projects/core/src/lib/layer-collections/layer-collection-navigation/layer-collection-navigation.component.ts

```diff
@@ -10,47 +10,96 @@ import {CONTEXT_TOKEN, LayerCollectionListComponent} from '../layer-collection-l
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class LayerCollectionNavigationComponent {
-    collections: Array<LayerCollectionItemDict | undefined> = [undefined];
+    collections: Array<LayerCollectionItemDict> = [];
+    allTrails: Array<Array<LayerCollectionItemDict>> = [];
+    displayedTrail: Array<LayerCollectionItemDict> = [];
 
-    selectedCollection = 0;
+    selectedCollection = -1;
 
     selectedPortal!: Portal<any>;
 
+    @ViewChild('scrollElement', {read: ElementRef}) public scrollElement!: ElementRef<any>;
+
     constructor() {
         this.setPortal(undefined);
     }
 
+    scrollToRight(): void {
+        setTimeout(() => {
+            // wait until breadcrumbs are re-rendered before scrolling
+            this.scrollElement.nativeElement.scrollLeft += Number.MAX_SAFE_INTEGER;
```

Comment:
this doesn't work, at least in firefox it doesn't scroll.

Reply:
Ideally, we could use the full width of the breadcrumb area here, as if it hadn't a scrollbar. But I can only find the width of the element after the scrollbar is applied, which is of course a fixed value and not large enough.

Although it's a magic number, I used 900000 now, which should be high enough in any case and tested it in Firefox and Safari as well.  0a65ceefb42d3a9b9bc603e0ea676a8a9eb358fb

Reply:
It works on Chrome, Firefox, Safari and Edge